spec/models/claims/financial_summary_spec.rb

```diff
@@ -10,22 +10,20 @@
     let!(:allocated_claim)  { create(:allocated_claim,) }
 
     let!(:old_part_authorised_claim) do
-      Timecop.freeze(Time.now - 2.week) do
-        claim = create(:part_authorised_claim)
-        Timecop.freeze(Time.now + 1.week) do
-          claim.determinations.first.update(fees: claim.fees_total/2, expenses: claim.expenses_total)
-          claim
-        end
+      travel_to(Time.now - 2.week)
+      create(:part_authorised_claim).tap do |claim|
+        travel_to(Time.now + 1.week)
```

Comment:
can/should we be using a block here

Reply:
I think that nesting `travel_to` can cause problems as the end of the block returns to the current time and not the previously frozen time. It is, in fact, calling `travel_back` after the `yield` but I thought that it would look clearer as:

```ruby
travel_to x
...
travel_to y
...
travel_back
```

although that was before I added `tap`.

I'm happy to add a block to the second `travel_to` (but nesting them, as they were before, is discouraged as it has complications).

Reply:
👍  makes sense, i'll leave it to you